### PR TITLE
Update create-dns-records-at-any-dns-hosting-provider.md

### PR DESCRIPTION
**Pull request by msbemba**
added another line to explain how to change the email address if the users were created before

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9593

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ At the end of these steps, your domain will stay registered with the host that y
 
 If you don't add a domain, people in your organization will use the onmicrosoft.com domain for their email addresses until you do. It's important to add your domain before you add users, so you don't have to set them up twice.
 
+If you have created users before and would like to change their domain, folow steps mentioned at [Change your email address to use your custom domain using the Microsoft 365 admin center](/admin/email/change-email-address?view=o365-worldwide#change-your-email-address-to-use-your-custom-domain-using-the-microsoft-365-admin-center)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider.md]:
```suggestion
If you have previously created users and would like to change their domain, follow the steps described in [Change your email address to use your custom domain using the Microsoft 365 admin center](/microsoft-365/admin/email/change-email-address#change-your-email-address-to-use-your-custom-domain-using-the-microsoft-365-admin-center).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**